cli/src/main/java/com/devonfw/tools/ide/common/SystemPath.java

```diff
@@ -155,16 +155,16 @@ public Path findBinary(Path toolPath) {
    * @return the {@link Path} to the directory of the tool where the binaries can be found or {@code null} if the tool
    *         is not installed.
    */
-  public Path getPath(String tool) {
+  public Path retrievePath(String tool) {
 
     return this.tool2pathMap.get(tool);
   }
 
   /**
    * @param tool the name of the tool.
-   * @param path the new {@link #getPath(String) tool bin path}.
+   * @param path the new {@link #retrievePath(String) tool bin path}.
    */
-  public void setPath(String tool, Path path) {
+  public void addPath(String tool, Path path) {
```

Comment:
Okey. Makes sense. I was talking to @jan-vcapgemini and we both came to the conclusion that it could be nice to only use the prefixes `set` and `get` with actual getters and setters.

Reply:
Adjusted, renamed back to get and set.